### PR TITLE
New Proof

### PR DESCRIPTION
**Pull request by theref**

### CODE REVIEW

File: _drafts/on_testing_degeneracy_of_games.md [hunk 1]
```diff
@@ -193,17 +193,35 @@ is sufficient to test.
...
 ## A sufficient mixed strategy to test for degeneracy
 
-Let \\(X\\) be the set of all strategies for Player1.
-Let \\(Y\\) be the set of all strategies for Player2.
+Given a support \\(\sigma\\) for Player 1, we want to find a strategy \\(x\\)
+with support \\(\sigma\\) that maximises the number of best respones for Player 2.
+If the game is degenerate then there will exist an \\(x\\) which has more best
+responses than the size of its support \\(\sigma\\).
+
+Here we show that for a degenerate game, it is sufficient to make Player 1 and
```
Comment (drvinceknight) [inline _drafts/on_testing_degeneracy_of_games.md]: This isn't quite what we want, they don't need to be indifferent to each other. We just need that for x makes player two indifferent for each size of support for player 2 and then subsequently (the algorithm might not even get here) that for y it makes player 1 indfferent for each of their supports.

**Comment by theref** [comment]
I forgot to remove the old commits, but the code gets deleted anyway

File: _drafts/on_testing_degeneracy_of_games.md [hunk 2]
```diff
@@ -193,18 +193,43 @@ is sufficient to test.
 
 ## A sufficient mixed strategy to test for degeneracy
 
-Let \\(X\\) be the set of all strategies for Player1.
-Let \\(Y\\) be the set of all strategies for Player2.
-For any \\(x \in X\\) let \\(S(x)\\) be the size of the support of \\(x\\).
-(Similar for any \\(y \in Y\\)).
+**Def.** A Normal Form Game is degenerate if:
```
Comment (drvinceknight) [inline _drafts/on_testing_degeneracy_of_games.md:196]: iff

(that's important here)

File: _drafts/on_testing_degeneracy_of_games.md [hunk 3]
```diff
@@ -193,18 +193,43 @@ is sufficient to test.
...
 
-If the game is degenerate, then there exists \\(x_* \in X\\) such that \\(S(x_*) = K\\) and Player2 has pure best responses \\(y_1, y_2,...,y_N\\) with \\(N > K\\).
-(By definition)
+\\( \exists \enspace x \\) s.t. \\( \mid S(x) \mid < \mid \sigma_2\mid \\)
+where \\( (xB)_j = \text{ Max}(xB) \\), \\( \forall j \\) in \\( \sigma_2 \\)
```
Comment (drvinceknight) [inline _drafts/on_testing_degeneracy_of_games.md:199]: not \text{Max} but \max

**Pull request merged**